src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -146,11 +146,16 @@
                             @if (hasSidebar) {
                                 <button
                                     [ngbTooltip]="'Toggle Sidebar (⇧ + Ctrl + B)'"
+                                    [ngClass]="isSidebarCollapsed ? 'is-collapsed' : ''"
```

Comment:
```suggestion
                                    [ngClass]="{'is-collapsed' : isSidebarCollapsed}"
```

Reply:
Thanks! In the last commits I fixed the hover animation and added the suggested change 👍🏼